modules/core/classes/GallerySmarty.class

```diff
@@ -42,6 +42,9 @@ class GallerySmarty extends Smarty {
      */
     var $_firstGalleryStatus;
 
+    /* deprecated dynamic properties in php 8.2 */
+    var $_cache_include_info;
```

Comment:
This doesn't seem to be needed.

Reply:
Without the variable declaration unit tests 33, 597, 751, 752, 759, 760, 766, 771, 772, 774, 776, 777, 779, 782, 1192 and another 20 or so unit tests fails with the deprecation warning from php 8.2